msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -1923,7 +1923,7 @@ private String saveTopicSettings(boolean draft)
     	}
     	
         topic.setBaseForum(selectedForum.getForum());
-        if(selectedForum.getForum().getRestrictPermissionsForGroups()){
+        if(selectedForum.getForum().getRestrictPermissionsForGroups() && !"true".equalsIgnoreCase(ServerConfigurationService.getString("msgcntr.unrestricted.group.perms"))){
```

Comment:
For all of these it would be better to use ServerConfigurationService.getBoolean. This is already used in this file. You'd also want to give it a default.